dao/src/main/scala/za/co/absa/enceladus/dao/menasplugin/MenasAuthUtils.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.dao.menasplugin
+
+import com.typesafe.config.ConfigFactory
+import org.apache.hadoop.conf.Configuration
+import sun.security.krb5.internal.ktab.KeyTab
+import za.co.absa.enceladus.utils.fs.FileSystemVersionUtils
+
+/**
+  * This object contins Menas authentication tools.
+  */
+class MenasAuthUtils(conf: Configuration) {
+  private val fsUtils = new FileSystemVersionUtils(conf)
+
+  /**
+    * Creates Menas credentials from a credentials file located either on local file system or on HDFS.
```

Comment:
```suggestion
    * Instantiates MenasCredentials from a credentials file located either on local file system or on HDFS.
```

Create Menas credentials seems like it will create a new user for you.

Reply:
Why `Menas credentials` -> `MenasCredentials`?

Reply:
Because it is the name of the abstract class, it can be `MenasPlainCredentials`